### PR TITLE
Add body

### PR DESCRIPTION
**Pull request by heliDevine**
Added main tag

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
-import "./App.css";
+import './App.css';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
+        <main>
+          <h1>Git Pull practice page</h1>
```
Comment (naylfc90) [inline src/App.js]:
```suggestion
          <h2>Git Pull practice page</h2>
```

**Review by naylfc90 — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
+          <p>Tuesday's lesson- breakout room 1</p>
```
Comment (Daveoxys) [inline src/App.js]:
```suggestion
  <h4>Git branching</h4>
  <p>Tuesday's lesson- breakout room 1</p>
```

**Review by Daveoxys — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 3]
```diff
@@ -1,9 +1,13 @@
-import "./App.css";
+import './App.css';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
+        <main>
```
Comment (Daveoxys) [inline src/App.js]: Main needs to be outside of the header.

**Review by Daveoxys — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 3 above)
Comment (heliDevine) [reply src/App.js]: thanks!!

**Pull request merged**